package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@harnessio/ff-javascript-client-sdk",
-  "version": "1.16.0",
+  "version": "1.17.0-rc.1",
```

The `rc` bit shouldn't be committed
```suggestion
  "version": "1.17.0",
```